docs/reference/package-control/client-troubleshooting.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Package Control Client: Troubleshooting"
+---
+
+<!-- Originals: -->
+<!-- https://packagecontrol.io/docs/troubleshooting -->
+<!-- https://github.com/wbond/packagecontrol.io/blob/master/app/html/docs/troubleshooting.html -->
+
+# Troubleshooting
+
+
+## There are no packages available for installation
+
+This message is displayed whenever Package Control can not find
+any packages that are installable.
+The most common reason for this is an HTTP error
+in downloading package information from the default channel.
+
+To help resolve the issue, follow these steps:
+
+1. Select the View _\>_ Show Console menu entry
+2. Look for any lines starting with "Package Control:"
+3. Try enabling the [debug log][debug] for more information
+4. Ensure any proxy information, if needed,
+   is set in the Package Control settings.
+5. If you have a proxy and it rewrites secure connections,
+   add its CA cert as trusted:
+    - Click the Preferences _\>_ Browse Packages… menu.
+    - Open the User folder.
+    - Create a file named Package Control.user-ca-bundle
+      and paste in a PEM-formatted version of the certificate.
+6. Make sure you have the latest version of Package Control installed.
+7. If you are still having trouble,
+   review the [open issues][issues].
+   If you do not find a relevant issue, please open a new one,
+   and be sure to include your debug log.
+
+[debug]: #Enabling_the_Debug_Log
+[issues]: https://github.com/wbond/package_control/issues?q=is%3Aopen
+
+
+## Purging and Reinstalling
+
+Many users run into issues with Package Control because they have an old,
+broken or pre-release version of Package Control that has become stuck.
+Since Package Control updates itself,
+a bug can prevent the automatic upgrade from working,
+leaving users orphaned on an old version.
+
+The first step in debugging issues with Package Control
+is to purge any existing copies.
+
+1. Select the Preferences _\>_ Settings – User menu entry.
+2. Remove "Package Control" from the "ignored\_packages" setting, if present.
```

Comment:
```suggestion
2. Remove "Package Control" from the `ignored_packages` setting, if present.
```

Not sure which way you're running these. Just make them consistent. Maybe the whole project should have a style guide?